docs/software/printing.md

```diff
@@ -2,11 +2,103 @@
 
 ## Overview
 
-!!! warning "Documentation in progress..." 
+- [**CloudPrint - CUPS print server, with support for Google cloud printing**](#cloudprint-cups-print-server)
+- [**OctoPrint - Web interface for controlling 3D printers**](#octoprint-web-interface-for-controlling-3d-printers)
 
-    The links redirect to the old documentation available on [DietPi.com Forum](https://dietpi.com/phpbb/viewtopic.php?f=8&t=5). **We need your help to create and update this documentation ! Thank you** :octicons-heart-16: :octicons-heart-16:
+??? info "How do I run **DietPi-Software** and install **Optimised software** ?"
+    To install any of the **DietPi Optimised Software** listed below run from the command line:
+    ```
+    dietpi-launcher
+    ```
+    and selected _DietPi-Software_. or launch directly the tool:
+    ```
+    dietpi-software
+    ```
 
-- [CloudPrint - CUPS print server, with support for Google cloud printing](https://dietpi.com/phpbb/viewtopic.php?f=8&t=5&start=100#p6630)  
-- [OctoPrint - Web interface for controlling 3D printers](https://dietpi.com/phpbb/viewtopic.php?p=7958#p7958)  
+    Choose **Software optimised** and select one or more items. Finally click on `Install`. DietPi will do all the necessary steps to install and start these software items.
+
+    ![DietPi software](../assets/images/dietpi-software.jpg)
+
+    To see all the DietPi configurations options, review [DietPi Tools](../../dietpi_tools) section.
 
 [Return to the **Optimised Software list**](../../dietpi_optimised_software)
+
+## CloudPrint - CUPS print server
+
+The package is based on the CUPS print server. It supports also the Google cloud printing.  
+Also included is a web interface for CUPS, allowing easy setup of printers.
+
+![DietPi print server software CUPS](../assets/images/dietpi-software-printserver-cups.png){: style="width:550px"}
+
+### Access the web interface
+
+The web interface is accessible via port 631 on the machine running the CUPS server, e.g. this could be:  
+
+- URL = `http://192.168.0.100:631`  
+- user = *root*  
+- password = *dietpi*
+
+### Add a printer
+
+In the CUPS web interface, you can add and configure printers by selecting:
+
+- *Home* \> *Adding Printers and Classes*
+
+### Enable CloudPrint
+
+Once the printer is setup you only have to run the following command to enable Google Cloud Print authentication on this device:
+
+    cps-auth
```

Comment:
This will produce a markdownlint error, since above ``` was used for code blocks, here four spaces. Let's stay with fenced code block consequently as this is more explicit (indentation is used in other cases as well) and allows to define the code language.

Reply:
Ok. I will change this.